source/_docs/installation.markdown

```diff
@@ -46,7 +46,7 @@ These install options are fully supported by Home Assistant's documentation. For
 
 <div class='note'>
 
-The only installation methods that allow you to use Home Assistant Add-ons is using the Home Assistant image. All other methods only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.
+The only installation methods that allow you to use Home Assistant Add-ons is using the [Home Assistant OS](/hassio/installation/)|[A supported platform](/hassio/installation/) image or a [Home Assistant Supervised](https://github.com/home-assistant/supervised-installer) install. All other methods only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.
```

Comment:
The supported installation images linked are also all Home Assistant OS, thus you are listing a duplicate?